templates/team.html

```diff
@@ -7,57 +7,38 @@
-      </div>
-    </div>
-
-    <div class="row row__center blue-background">
-      <div class="column centered">
-          <h2><a href="https://www.linkedin.com/in/talea-carpenter/" target="_blank" rel="noopener noreferrer">TaLea Carpenter, <br>Associate Program Manager</a></h2>
-          <p class="justify">Before learning to code, TaLea spent the majority of her time homeschooling her children and making music. She has been recording music for over a decade and sees coding as another creative outlet. She worked at Ancestry following her training at Techtonica and is happy to be in a position to empower others.</p>
-      </div>
-      <div class = "column centered">
-        <img src="{{ url_for('static', filename='img/TaLea-min.jpg') }}"
-            alt="TaLea Carpenter"
-            class="resize-400px">
-      </div>
-    </div>
-
-    <div class="row row__center blue-background">
-      <div class = "column centered">
-        <img src="{{ url_for('static', filename='img/Tiffany-Quach-min.jpg') }}"
-            alt="Tiffany Quach"
-            class="resize-400px">
-      </div>
-      <div class="column centered">
-          <h2><a href="https://linkedin.com/in/tiffanyaquach/" target="_blank" rel="noopener noreferrer">Tiffany Quach, <br>Partnership Development Manager</a></h2>
-          <p class="justify">Tiffany has over 10 years of professional experience working in social impact. Prior to joining Techtonica, she managed a STEM and Environmental Education program for youth from communities that organizations have previously failed to engage. She is an experienced community builder and is impassioned about diversity, equity, and inclusion across all sectors. She is now excited to help create an inclusive tech ecosystem through partnership and relationship building.</p>
+    {% for employee in team['employees'] %}
+      <div class="row row__center blue-background row--alternatingColumns">
+        <div class = "column centered">
+          <img src="{{ url_for('static', filename='img/%s' % employee['photo_filename'] ) }}"
+              alt="{{ employee['name'] }}"
+              class="resize-400px">
+        </div>
+        <div class="column centered">
+            <h2>
+              <a href="{{ employee['link_url'] }}" target="_blank" rel="noopener noreferrer">
+                {{ employee['name'] }}
+              </a>
+            </h2>
+            <h3>
+              <a href="{{ employee['link_url'] }}" target="_blank" rel="noopener noreferrer">
```

Comment:
Having two links to the same place next to each other is not good for accessibility.